packages/wonder-blocks-dropdown/components/multi-select.md

```diff
@@ -258,7 +258,7 @@ const modalContent = (
 );
 
 const modal = (
-    <StandardModal
+    <OnePaneDialog
```

Comment:
The unfortunate thing about this change is that we lose old docs.  I think we should post a link to one of the old netlify deploys along with a message along the lines of "Looking for docs for StandardModal, OneColumnModal, or TwoColumnModal click here".